brickschema/validate.py

```diff
@@ -0,0 +1,423 @@
+"""
+The `validate` module implements a wrapper of `pySHACL`_ to
+validate an ontology graph against the default Brick Schema constraints (called *shapes*) and user-defined shapes.
+
+.. _`pySHACL`: https://github.com/RDFLib/pySHACL
+"""
+import sys
+import argparse
+import logging
+from rdflib import Graph, Namespace, URIRef, BNode, Literal
+from rdflib.plugins.sparql import prepareQuery
+from .namespaces import BRICK, A, RDF, RDFS, BRICK, BSH, SH, SKOS, bind_prefixes
+import pyshacl
+import io
+import pkgutil
+
+class Validate():
```

Comment:
Class should be renamed `Validator` as this is grammatically consistent with what instances of this class are doing.